docker-compose.yml

```diff
@@ -134,8 +134,8 @@ services:
       - back
 
   indexer:
-    build: ./indexer/0.0.9-SNAPSHOT
-    image: oapass/indexer:0.0.9-2.1-SNAPSHOT
+    build: ./indexer/0.0.10-SNAPSHOT
+    image: oapass/indexer:0.0.10-2.1-SNAPSHOT
```

Comment:
`oapass/indexer:0.0.10-2.2-SNAPSHOT`

Also, it would be helpful to bump the tag of `fcrepo` in a corresponding way (2.1 -> 2.2)